README.md

```diff
@@ -2,7 +2,9 @@
 
 <a href="http://www.ohdsi.org/web/atlas"><img src="http://www.ohdsi.org/web/wiki/lib/exe/fetch.php?cache=&media=documentation:software:logo_without_text.png" align="left" hspace="10" vspace="6" width="164" height="200"></a>
 
-**ATLAS** is an open source software tool for researchers to conduct scientific analyses on standardized observational data converted to the [OMOP Common Data Model V5](https://github.com/OHDSI/CommonDataModel/wiki "OMOP Common Data Model V5"). Researchers can create cohorts by defining groups of people based on an exposure to a drug or diagnosis of a particular condition using healthcare claims data. ATLAS has vocabulary searching of medical concepts to identify people with specific conditions, drug exposures etc. Patient profiles can be viewed within a specific cohort allowing visualization of a particular subject's health care records. Population effect level estimation analyses allows for comparison of two different cohorts and leverages R packages.
+**ATLAS** is an open source software tool for researchers to conduct scientific analyses on patient-level healthcare data standardized to the [OMOP Common Data Model V5](https://ohdsi.github.io/CommonDataModel/ "OMOP Common Data Model V5"). Researchers can create cohorts by defining groups of people based on an exposure to a drug or diagnosis of a particular condition using healthcare claims data. ATLAS has vocabulary searching of medical concepts to identify people with specific conditions, drug exposures etc. More information on ATLAS is found in [Chapter 8 in the Book of OHDSI](https://ohdsi.github.io/TheBookOfOhdsi/OhdsiAnalyticsTools.html).
+
+The vision for ATLAS is to provide a common set of study design tools that are capable of running one or more [HADES](https://ohdsi.github.io/Hades/) analytical R packages.

```

Comment:
Another vision/thought on this: ATLAS is a design tool that supports the OHDSI communities standards & best practices for designing observational studies.